df.md

```diff
@@ -0,0 +1,49 @@
+# 🦈 Senior Product Developer - Data Farming 
+[Ocean Protocol](https://www.oceanprotocol.com) builds tools for people to reclaim control of their data and unleash an open data economy. Ocean tools make it easy to publish & consume data services as data access tokens, and bridge data to DeFi. 
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
+
+
+## Responsibilities
+* Candidate will have to communicate with different teams to sync on roadmap delivery
+* Candidate will need to engage with the community to get faster feedback to understand usage patterns, pain points, and user experience
+* They will need to understand to differentiate good and bad feedback and find real pain points and needed improvements
+* They will improve and evolve product/program based on learnings from usage data and user feedback 
+* They will take ownership of related tech like scripts and manage and update scripts and reward functions to implement design changes and effectively communicate to the community about changes
+* They will have to execute scripts to automate some web3 tasks like token distribution
+* They will have to identify fraudulent actors taking unfair advantage of the program and make improvements to curb them
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
+
+
+## Skills and Experience 
+* We don’t expect applicants to be an expert in everything listed below which is why we’ve separated the requirements into two tiers. The ideal candidate loves to communicate, to code, to build meaningful technical communities and relationships with key stakeholders.
+
+
+## Minimum Qualifications
+* 3+ years of experience working in technical/development roles in the crypto projects
+* 2+ years Hands-on experience in development in Web3 and Web2 tech stack
```

Comment:
My comments are for the two bullets above: 3+ years exp crypto, 2+ years dev Web 2 & Web3

Problems:
- 3+ years working in Web3 is a big ask; it greatly narrows our candidates. Same for "2+ years working in Web3 development". It's unnecessarily constraining.
- Web2 experience is a radically different thing than Web3, it should be its own line.
- Web2 and Web3 it should be more specific on which stack. 
- Having two separate lines on Web3 is a bit confusing. If you ask for a min req on Solidity experience you don't need a separate line on more general "technical" Web3 experience

Suggested new lines:

- 6+ months Solidity experience
- 3+ years full-stack Web2 experience including NodeJS